src/shared/components/palette/base-field/field-read.component.ts

```diff
@@ -28,19 +28,19 @@ export class FieldReadComponent extends AbstractFieldReadComponent implements On
   }
 
   ngOnInit(): void {
-    let componentClass = this.paletteService.getFieldComponentClass(this.caseField, false);
-    let injector = Injector.create([], this.fieldContainer.parentInjector);
-    let component = this.resolver.resolveComponentFactory(componentClass).create(injector);
-
-    // Provide component @Inputs
-    component.instance['caseField'] = plainToClassFromExist(new CaseField(), this.caseField);
-    component.instance['caseFields'] = this.caseFields;
-    component.instance['formGroup'] = this.formGroup;
-    component.instance['caseReference'] = this.caseReference;
-    component.instance['context'] = this.context;
-    component.instance['registerControl'] = this.registerControl || this.defaultControlRegister();
-
-    this.fieldContainer.insert(component.hostView);
+    Promise.resolve(null).then(() => {
```

Comment:
what was the issue? Perhaps we can fix it by using a more appropriate life cycle hook?
https://angular.io/guide/lifecycle-hooks

Reply:
The issue is we need the interpolated value inside the ngOnInit to build the field but the value is not there yet. The angular lifecycle kicks in before the label interpolation resolves the placeholder so we need to let it wait a cycle for the already resolved placeholder.